src/helpers/profile.ts

```diff
@@ -60,14 +61,22 @@ async function udReverseRecordRequest(addresses) {
 async function lookupAddresses(
   addresses: string[]
 ): Promise<{ [k: string]: string }> {
-  const [ens, ud] = await Promise.all([
+  const [ens, ud, lens] = await Promise.all([
```

Comment:
Naming for `getEnsAddress` doesn't seem right. Why is that function still even needed?